notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>Is there a reason to use TensorFlow 2.3? If not, please use a recent version of TensorFlow container image. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>No reason. I used 2.8 instead and it still works.</p>